projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -104,6 +104,11 @@ function () {
 				My_Jetpack_Initializer::init();
 			}
 		);
+		add_action(
+			'init',
+			array( new Automattic\Jetpack\Social\Jetpack_Social_Note(), 'register' ),
+			1
```

Comment:
Is there a reason to make this high priority?

Reply:
I have changed it, it works with the default priority as well.